lib/schema/schemaEnv.js

```diff
@@ -25,10 +25,21 @@ module.exports = (function () {
 
   // The version that we will accept upload requests from, will be in format of 0.200.0
   // NOTE: This will probably be eliminated in favor of using schemaVersion above for the same purpose
-  schemaEnv.minimumUploaderVersion = config.fromEnvironment('MINIMUM_UPLOADER_VERSION', '2.53.0');
+  schemaEnv.minimumUploaderVersion = config.fromEnvironment(
+    'MINIMUM_UPLOADER_VERSION',
+    '2.53.0'
+  );
 
   schemaEnv.authRealm = config.fromEnvironment('KEYCLOAK_AUTH_REALM', null);
   schemaEnv.authUrl = config.fromEnvironment('KEYCLOAK_AUTH_URL', null);
+  var usersStr = config.fromEnvironment('UPLOADER_PLATFORM_USER_IDS', null);
+  var platformUsers = [];
+  if (usersStr) {
+    platformUsers = usersStr
+      .split(',')
+      .map((item) => item.toLowerCase().trim());
+  }
+  schemaEnv.platformUserIds = platformUsers;
```

Comment:
To simplify, perhaps:
```suggestion
  schemaEnv.platformUserIds = config.fromEnvironment('UPLOADER_PLATFORM_USER_IDS', '')
      .split(',')
      .map((item) => item.toLowerCase().trim());
```

Reply:
I have updated to pull platform encrypted  userIds from file per environment and throw an error if the file doesn't exist